### PR TITLE
Add/modify resource descriptions

### PR DESCRIPTION
**Pull request by jenssegers**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -433,6 +433,8 @@ Get details for a single user.
 
 ## Custom Fields [/customFieldDefinitions]
 
+Custom fields are used to add additional data/properties to entities within Teamleader.
```
Comment (WouterSioen) [inline apiary.apib:436]: Q: `entities` might be a bit of a technical term here? Wouldn't something like "other objects" be clearer?

It's API documentation though, so understanding what is meant with entities might not be an issue.

File: apiary.apib [hunk 2]
```diff
@@ -1134,6 +1137,8 @@ Get the names of business types (legal structures) a company can have within a c
 
 ## Tags [/tags]
 
+Tags are used to label contacts or companies, so that they can be easily filtered or grouped into segments.
```
Comment (WouterSioen) [inline apiary.apib]: PP: `so that they can easily be filtered` reads a bit better imo (swap easily and be)

File: apiary.apib [hunk 3]
```diff
@@ -1166,6 +1171,8 @@ Get a list of tags.
 
 ## Deals [/deals]
 
+Deals are sales opportunities, which need to be followed up by their responsible user. A deal can move through specific deal phases, and be marked as won or lost.
```
Comment (WouterSioen) [inline apiary.apib]: `sale opportunities` (singular)

File: apiary.apib [hunk 4]
```diff
@@ -1437,7 +1444,9 @@ Get a list of lost reasons for deals.
                 + id: `6e7fe84d-d4b3-4723-abae-9bc082d8f65a` (string)
                 + name: `Deal declined` (string)
 
-## Deal Phases [/]
+## Deal Phases [/dealPhases]
+
+Deal phases are the different phases a deal typically goes through. A deal can be moved to either a next or previous phase. A deal can be marked as won or lost independently from the deal phase.
```
Comment (WouterSioen) [inline apiary.apib:1449]: `A deal can be marked as won or lost independently from the deal phase.`

That's not true (yet)

File: apiary.apib [hunk 5]
```diff
@@ -2305,8 +2316,7 @@ Get a list of product categories.
 
 ## Projects [/projects]
 
-Projects allow users to work together as a team on a single project.
-They comprise of one or more milestones, have one or more participants and are limited in time and budget.
+Projects allow users to work together as a team on a single project. They comprise of one or more milestones, have one or more participants and are limited in time and budget.
```
Comment (WouterSioen) [inline apiary.apib]: `comprise` has a bit of a bad connotation (or that's at least how it's feels to me). I think `consists of` might be better here?

File: apiary.apib [hunk 6]
```diff
@@ -2632,6 +2641,8 @@ Update a milestone.
 
 ## Time Tracking [/timeTracking]
 
+Time tracking gives you insight into the amount of work done and the work type. You are able to follow-up on the budgets of your projects and eventually you are able to invoice the billable time.
```
Comment (WouterSioen) [inline apiary.apib]: `and the work type` might be better as `and its work type`

Comment (WouterSioen) [reply apiary.apib]: > There is a space between follow and up under one condition—you are using follow up as a verb. To follow up means to gather further information or to reinforce or evaluate a previous action.

File: apiary.apib (hunk 5 above)
Comment (jenssegers) [reply apiary.apib]: :ok:

File: apiary.apib (hunk 3 above)
Comment (jenssegers) [reply apiary.apib]: :ok:

File: apiary.apib (hunk 2 above)
Comment (jenssegers) [reply apiary.apib]: :ok:

File: apiary.apib (hunk 6 above)
Comment (jenssegers) [reply apiary.apib]: I asked Niels to improve this text.

File: apiary.apib [hunk 7]
```diff
@@ -2305,8 +2316,7 @@ Get a list of product categories.
 
 ## Projects [/projects]
 
-Projects allow users to work together as a team on a single project.
-They comprise of one or more milestones, have one or more participants and are limited in time and budget.
+Projects allow users to work together as a team on a single project. They consists of one or more milestones, have one or more participants and are limited in time and budget.
```
Comment (WouterSioen) [inline apiary.apib]: `they consist of` (singular) 🙈

Comment (jenssegers) [reply apiary.apib]: :ok:

File: apiary.apib (hunk 4 above)
Comment (WouterSioen) [reply apiary.apib:1449]: @jenssegers I don't think this should've been merged with this information in it since this is incorrect...

Comment (jenssegers) [reply apiary.apib:1449]: @woutersioen I'll check with Niels for a new version!

**Pull request merged**